src/firebolt/async_db/connection.py

```diff
@@ -17,6 +20,8 @@
 from firebolt.common.util import fix_url_schema
 
 DEFAULT_TIMEOUT_SECONDS: int = 5
+KEEPALIVE_FLAG: int = 1
+KEEPIDLE_RATE: int = 60  # seconds
```

Comment:
do I understand correctly, as long as this value is less than 350, the trick will work, right?

Reply:
Technically yes, I was trying to find a typical expected value for this but it differs a lot. Wiki gives 45-60 range so I went with the latter.